code/datums/controllers/sea_hotspot_controls.dm

```diff
@@ -39,42 +39,47 @@
 
 	#ifdef UNDERWATER_MAP
 	var/list/map_colors = list(
-		empty = rgb(0, 0, 50),
-		solid = rgb(0, 0, 255),
+		empty = rgb(0, 0, 255),
+		solid = rgb(0, 0, 50),
 		station = rgb(255, 153, 58),
-		other = rgb(120, 200, 120))
+		other = rgb(120, 200, 120),
+		transparant = null)
 	#else
 	var/list/map_colors = list(
 		empty = rgb(30, 30, 45),
 		solid = rgb(180,180,180),
 		station = rgb(27, 163, 186),
-		other = rgb(186, 0, 60))
+		other = rgb(186, 0, 60),
+		transparant = null)
 	#endif
 
 	proc/generate_map()
 		if (!map)
 			Z_LOG_DEBUG("Mining Map", "Generating map ...")
 			map = icon('icons/misc/trenchMapEmpty.dmi', "template")
 			var/turf_color = null
+			var/size_mult = 2
 			for (var/x in 1 to world.maxx)
 				for (var/y in 1 to world.maxy)
 					var/turf/T = locate(x,y,MINING_Z)
 					if (istype(T, /turf/simulated/wall/auto/asteroid) || istype(T, /turf/simulated/floor/plating/airless/asteroid))
 						turf_color = "solid"
 					else if (istype(T, /turf/space))
-						turf_color = "empty"
+						turf_color = "transparant"
```

Comment:
Why even bother drawing then? Why not just ignore processing this?

Reply:
I definitely overlooked that, I just committed a corrected version that is much simpler